src/ex1/styles/buttons.css

```diff
@@ -0,0 +1,50 @@
+@import "button-hover.css";
+
+button {
+  cursor: pointer;
+}
+
+#add-todo-button {
+  background-color: var(--dark-green);
+  border-radius: 4px;
+  border: 0px;
```

Comment:
some of the styles here could be moved to a baseline .button class to share a common style between all the buttons

Reply:
done